Source/WebCore/platform/graphics/cocoa/MediaPlaybackTargetContext.h

```diff
@@ -57,21 +56,46 @@ class MediaPlaybackTargetContext {
     MediaPlaybackTargetContext() = default;
     WEBCORE_EXPORT explicit MediaPlaybackTargetContext(RetainPtr<AVOutputContext>&&);
 
-    MediaPlaybackTargetContext(RetainPtr<NSData>&& serializedOutputContext, bool hasActiveRoute)
-        : m_type(Type::SerializedAVOutputContext)
-        , m_serializedOutputContext(WTFMove(serializedOutputContext))
-        , m_cachedHasActiveRoute(hasActiveRoute)
-    {
-        ASSERT(m_serializedOutputContext);
-    }
-
     MediaPlaybackTargetContext(const String& mockDeviceName, MockState state)
         : m_type(Type::Mock)
         , m_mockDeviceName(mockDeviceName)
         , m_mockState(state)
     {
     }
 
+#if PLATFORM(COCOA)
```

Comment:
This doesn't make sense, we're already in a cocoa/ folder. We shouldn't need this.

Reply:
Fixed